karate-tests/src/test/java/mocks/sandbox/get-patient-search.js

```diff
@@ -149,14 +150,20 @@ if (request.pathMatches('/Patient') && request.get) {
 
   if (validateHeaders(request) && validateQueryParams(request)) {
     if (fuzzyMatch) {
-      if (family === 'Garton' && (given[0]) === 'Bill' && birthDate[0] === '1946-06-23') {
-        response.body = timestampBody(BILL_GARTON_SEARCHSET)
+      if (family === 'Blogs' && given[0] === 'Joe' && birthDate[0] === '1955-11-05') {
+        response.body = timestampBody(JOE_BLOGS_HISOTRIC_NAME_SEARCHSET)
       } else if (family === 'ATTSÖN' && (given[0]) === 'PÀULINÉ' && birthDate[0] === '1960-07-14') {
         response.body = timestampBody(PAULINE_ATTISON_SEARCHSET)
       } else if (phone === '01222111111' && email === 'test@test.com') {
         response.body = timestampBody(FUZZY_SINGLE_SEARCHSET)
       } else if (family === 'Smythe' && (given[0]) === 'Mat' && (birthDate[0]) === 'ge2000-05-03' && gender === 'male' && postalCode === 'DN17 4AA' && email !== 'rubbish@work.com') {
         response.body = timestampBody(FUZZY_MULTI_SEARCHSET)
+      } else if (['MED', 'HUME'].includes(family) && (given[0]) === 'Casey' && (birthDate[0]) === '1999-09-09') {
+        response.body = timestampBody(HISTORIC_DATA_SEARCHSET)
+      } else if (family === 'MED' && (given[0]) === 'Casey' && (birthDate[0]) === '2024-01-12') {
+        response.body = timestampBody(EMPTY_SEARCHSET)
+      } else if (family === 'LEEKE' && (given[0]) === 'Horace' && (birthDate[0]) === '1956-05-02' && postalCode === 'DN16') {
+        response.body = timestampBody(EMPTY_SEARCHSET)
```

Comment:
Is there a better way to set the response.body, other than loads of "else if"s ? Perhaps some dictionary that we can loop through?

Reply:
I will create dedicated technical debt ticket and  refactor existing else if structures using switch or dictionary